_topic_maps/_topic_map.yml

```diff
@@ -193,13 +194,18 @@ Topics:
     File: domain-mapping-odc-admin
   - Name: Securing a mapped service using a TLS certificate
     File: domain-mapping-custom-tls-cert
-- Name: Configuring high availability for Knative services
+- Name: High availability configuration for Knative Serving
   Dir: config-ha-services
   Topics:
   - Name: High availability for Knative services overview
     File: ha-services-overview
-  - Name: High availability for Knative services
+  - Name: High availability for Knative deployments
     File: ha-replicas-serving
+- Name: Tuning serving configuration 
+  Dir: tuning-serving-configuration
+  Topics:  
+  - Name: Overriding system deployment configurations
```

Comment:
While we're moving this document, we should fix one additional issue. There's a note about net-istio and net-kourier missing liveness or/and readiness probes: https://68373--docspreview.netlify.app/openshift-serverless/latest/knative-serving/tuning-serving-configuration/overriding-config-serving
![image](https://github.com/openshift/openshift-docs/assets/336823/ec1c7a99-634a-45b0-9478-15424feca795)

This not should be removed because it was fixed by https://issues.redhat.com/browse/SRVKS-1046

Reply:
Done! Removed the note